pipelines/controller/src/main/resources/templates/index.html

```diff
@@ -135,6 +135,22 @@
       failure.style.display = "none";
     }, 4000);
 
+    function fetchLatestMetrics() {
+      var xhr = new XMLHttpRequest();
+      xhr.open("GET", '/actuator/prometheus', true);
```

Comment:
How many metrics do we get here? I mean it seems it is not only our pipeline metrics and I wonder whether we should do any clean-up of the metrics to make more relevant values easier to find.

Reply:
There are many metrics that gets logged, this [doc](https://docs.google.com/document/d/15Gnmg5qrKdtoR1vgKjKmlurLeaHmGMUuZo30xm6PZUo/edit?resourcekey=0-jKdcuYogcB4ZCJ8oz2iQUg) contains a sample.  In the latest PR, I have made changes to expose only the pipeline metrics (snapshot attached in the PR description)

Reply:
Okay, so I think all of these are still accessible on the default Prometheus endpoint, correct? And with your recent changes we only expose our pipeline specific metrics in our UI which is great; resolving this.